bbpower/compsep.py

```diff
@@ -112,6 +113,7 @@ def parse_sacc_file(self):
 
         # Read data
         self.s = sacc.Sacc.load_fits(self.get_input('cells_coadded'))
+        self.s_cov = sacc.Sacc.load_fits(self.get_input('cells_coadded_cov'))
```

Comment:
could we add a check that the two files have the same ordering and throw an error otherwise?
This could be done with something like:
```
tr_comb = self.s.get_tracer_combinations()
for tr1, tr2 in tr_comb:
    ind1 = self.s.indices(data_type='cl_bb', tracers=(tr1, tr2))
    ind2 = self.s_cov.indices(data_type='cl_bb', tracers=(tr1, tr2))
    assert np.all(ind1 == ind2)
```

Reply:
Done!